src/main/kotlin/me/serce/solidity/ide/settings/SolidityConfigurablePanel.kt

```diff
@@ -0,0 +1,46 @@
+package me.serce.solidity.ide.settings
+
+import com.intellij.openapi.fileChooser.FileChooserDescriptorFactory
+import com.intellij.openapi.ui.TextFieldWithBrowseButton
+import com.intellij.openapi.util.io.FileUtil
+import javax.swing.JPanel
+
+class SolidityConfigurablePanel {
+  private var myEvmPathField: TextFieldWithBrowseButton? = null
+  private var myDbPathField: TextFieldWithBrowseButton? = null
```

Comment:
I assume that `lateinit` can be used here to avoid all nullability problems and preserve bindings, that way all `!!` should disappear

Reply:
Oh, thank you for pointing me this feature!
Done